.golangci.yml

```diff
@@ -3,3 +3,5 @@ linters-settings:
     line-length: 170
   dupl:
     threshold: 400
+run:
+  deadline: 5m
```

Comment:
`deadline` is deprecated... we should switch to `timeout` 

details: https://github.com/golangci/golangci-lint/pull/822